packages/angular_devkit/build_angular/src/builders/application/build-action.ts

```diff
@@ -164,6 +167,10 @@ export async function* runEsBuildBuildAction(
         break;
       }
 
+      if (clearScreen) {
+        clearTerminal();
```

Comment:
```suggestion
        console.clear();
```
It essentially does what this call is doing.